packages/orchestration/src/exos/local-chain-facade.js

```diff
@@ -54,8 +54,8 @@ export const prepareLocalChainFacade = (
       makeAccount() {
         const { localChainInfo } = this.state;
         const lcaP = E(localchain).makeAccount();
-        // FIXME use watch() from vowTools
-        return heapVowTools.watch(allVows([lcaP, E(lcaP).getAddress()]), {
+        // @ts-expect-error E does not understand Vow pipelining
```

Comment:
That seems like a devex issue that our customers may encounter.

IOU an issue. Bonus points if you beat me to it.
cc @Jovonni @mitdralla

Reply:
>  E does not understand Vow pipelining

I thought this was the case, but it doesn't seem to be. I was tricked by the linter at one point and this comment is not in the final revision.

Reply:
On further analysis, the original comment does appear to be true. In this particular instance, `E(localchain).makeAccount` is a Promise, not a Vow, so pipelining works.

Here's a test that demonstrate the behavior: [54ef388](https://github.com/Agoric/agoric-sdk/pull/9591/commits/54ef388c8c4c681866f228622e3294f70c747858)